eng/scripts/CodeChecks.ps1

```diff
@@ -100,8 +100,8 @@ try {
             $status = $status -replace "`n","`n    "
             LogError "Generated code is not up to date.`n" + `
                 "You may need to rebase on the latest master, `n" + `
-                "run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files, `n" + `
-                "run 'eng\scripts\Export-API.ps1' if you changed public APIs `n" +
+                "run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files (https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets), `n" + `
```

Comment:
I didn't realize there were individual links to these different pieces when I was giving my original feedback I was thinking there would be a single link for this entire error. However I think this is also reasonable as well if folks need to better understand those scripts.